### PR TITLE
FEAT-43- Adding FAQ 

### PR DESCRIPTION
**Pull request by twentymurial33**
## Change Type
*Put an `x` in all the boxes that apply
- [] Bugfix (non-breaking change which fixes an issue)
- [X] Feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to change)

## Linear issue link
https://linear.app/oku-no-kumo/issue/OKU-44/faq-component

## Before & after screenshots or video recordings (Before & After)
Screenshot 2023-08-14 at 2 10 56 PM

### CODE REVIEW

**Review by idiglove — COMMENTED** [review]
this is supposed to be an Accordion, we need to fix it. 
Also link it to the footer link

File: project_app/components/FaqTest.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+export default function BasicAccordion() {
+  return (
+    <AccordianWrapper>
+      <Accordion style={{backgroundColor:"rgb(53, 53, 53)",width:"50%",marginLeft:"300px",border: "2px solid white"}}>
+        <AccordionSummary
+       expandIcon={<ExpandMoreIcon style={{color:"white"}}/>}
+        >
+        <Typography>FAQ 1</Typography>
+        </AccordionSummary>
+      </Accordion>
```
[the remark this answers is not in the document]
Comment (twentymurial33) [reply project_app/components/FaqTest.tsx:16]: done

File: project_app/components/Accordion.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          marginLeft: '300px',
+          border: '2px solid white'
+        }}
+      >
+        <AccordionSummary
+          expandIcon={<ExpandMoreIcon style={{ color: 'white' }} />}
+        >
+          <Typography>FAQ 1</Typography>
+        </AccordionSummary>
+      </Accordion>
```
Comment (idiglove) [inline project_app/components/Accordion.js:22]: this is repeating. there's a pattern, this in itself should be a reusable component

Comment (twentymurial33) [reply project_app/components/Accordion.js:22]: Oh got it - working on it

**Review by idiglove — APPROVED** [review]
better .lgtm

**Pull request merged**